doc/source/coding-style/formatting-tools.rst

```diff
@@ -0,0 +1,241 @@
+Code Style Tools
+================
+
+There are plenty of tools for checking code style. This section presents some of
+the most popular ones in the Python ecosystem. A minimum configuration is
+provided for each one so you can easily include them in your PyAnsys project.
```

Comment:
```suggestion
There are many tools for checking code style. This section presents some of
the most popular ones in the Python ecosystem. A minimum configuration is
provided for each one so that you can easily include them in your PyAnsys project.
```